src/routing/approved.rs

```diff
@@ -1158,6 +1176,36 @@ impl Approved {
         let age = peer.age();
         let signature = proof.signature.clone();
 
+        // A vote of half-aged Left peer means such peer is trying to rejoin with the same name,
+        // and shall be relocated immediately. The flow will return early during the process of
+        // update_member, as `Left -> Relocated` is not allowed.
+        //
+        // TODO: shall we make `Left -> Relocated` to avoid the left node is being repeatedly used
+        //       to generate higher new nodes?
+        if let Some(info) = self.section.members().get(member_info.peer.name()) {
+            if info.peer.age() > age {
+                let variant = Variant::Relocate(RelocateDetails::new(
+                    &self.section,
+                    &self.network,
+                    &peer,
+                    *peer.name(),
+                ));
+                trace!("Send {:?} to {:?}", variant, peer);
+                let message = Message::single_src(
+                    &self.node,
+                    DstLocation::Direct,
+                    variant,
+                    Some(self.section.create_proof_chain_for_our_info(None)),
+                    None,
+                )?;
+
+                commands.push(Command::send_message_to_target(
+                    peer.addr(),
+                    message.to_bytes(),
+                ));
```

Comment:
The `Relocate` message cannot be sent like this, it needs to be accumulated. Just use `send_relocate` instead of this.

Reply:
`send_relocate ` requires the receiver to have a vote_accumulator, which a `bootstraping` node doesn't have yet?